#! /bin/sh /usr/share/dpatch/dpatch-run
## 015_disable_expat_buildconf.dpatch by Stefan Fritsch <sf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: We don't need the bundled expat. This allows us to avoid a build-dep
## DP: on libtool.

@DPATCH@
diff -urNad trunk~/buildconf trunk/buildconf
--- trunk~/buildconf	2009-09-12 14:45:13.892735207 +0200
+++ trunk/buildconf	2009-09-12 14:45:46.560734181 +0200
@@ -84,14 +84,6 @@
 echo "Generating 'make' outputs ..."
 $apr_src_dir/build/gen-build.py $verbose make
 
-#
-# If Expat has been bundled, then go and configure the thing
-#
-if [ -f xml/expat/buildconf.sh ]; then
-  echo "Invoking xml/expat/buildconf.sh ..."
-  (cd xml/expat; ./buildconf.sh $verbose)
-fi
-
 # Remove autoconf cache again
 rm -rf autom4te*.cache
 
